<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>425</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>325</width>
    <height>425</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>325</width>
    <height>425</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widgetPrincipal" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: #F7F9FC;
color: #2d2d2d;
}
QLineEdit {
    border: 2px solid #2980b9;
    border-radius: 12px;
    padding: 3px 8px;
    background-color: white;
	color: #2d2d2d;
}</string>
     </property>
     <widget class="QLabel" name="lblTitulo_login">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>325</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: #2c3e50; 
    qproperty-alignment: AlignCenter;
    margin-bottom: 10px;
}</string>
      </property>
      <property name="text">
       <string>Sistema de Registro Académico</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblLogo_login">
      <property name="geometry">
       <rect>
        <x>114</x>
        <y>110</y>
        <width>80</width>
        <height>90</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../icons/resources_ui.qrc">:/logos/logo_escuela_sinFondo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputUser">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>241</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>270</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::StrongFocus</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>USUARIO</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputPassword">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>241</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>241</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::StrongFocus</enum>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>CONTRASEÑA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLogin">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>340</y>
        <width>200</width>
        <height>35</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::ClickFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #2980b9;
    color: #FFFFFF;
    border: none;
    padding: 8px 10px;
    border-radius: 15px;
}
QPushButton:hover {
	background-color: #0D47A1
}</string>
      </property>
      <property name="text">
       <string>Acceder</string>
      </property>
      <property name="icon">
       <iconset resource="../icons/resources_ui.qrc">
        <normaloff>:/icons/login_w.png</normaloff>:/icons/login_w.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="lblTitulo_login_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>241</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>307</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {

    color: #2c3e50; 
    qproperty-alignment: AlignCenter;
    margin-bottom: 10px;
    border-bottom: 2px solid #003366; /* línea decorativa */
}</string>
      </property>
      <property name="text">
       <string>E.B. &quot;Dr. Severiano Hernández&quot;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>lblLogo_login</zorder>
     <zorder>inputUser</zorder>
     <zorder>inputPassword</zorder>
     <zorder>btnLogin</zorder>
     <zorder>lblTitulo_login_2</zorder>
     <zorder>lblTitulo_login</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/resources_ui.qrc"/>
  <include location="../icons/resources_ui.qrc"/>
 </resources>
 <connections/>
</ui>
